src/error.rs

```diff
@@ -0,0 +1,25 @@
+//! Contains error types and other error handling tools.
+
+pub use crate::parse::ParseIntError;
+
+/// Impls std::error::Error for the specified type with appropriate attributes, possibly returning
+/// source.
+#[macro_export]
+macro_rules! impl_std_error {
+    // No source available
```

Comment:
```suggestion
    // Field is not an error source.
```

Reply:
Just a nit. My aim was to make the macro more approachable for devs new to our error code boiler plate code and/or macros in general.

Reply:
There could also be no field. :) Maybe "No field is an error source"?